test/e2e/app-dir/metadata-dynamic-routes/app/layout.tsx

```diff
@@ -8,7 +8,6 @@ export default function Layout({ children }) {
 }
 
 export const metadata = {
-  metadataBase: new URL('https://mydomain.com'),
```

Comment:
This was removed so we can assert on the automatic metadataBase detection in the test cases. ie, when deployed, it's a Vercel URL. Otherwise it's localhost.